browser/js/cart/cart.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+app.config(function ($stateProvider) {
+    $stateProvider.state('cart', {
+        url: '/cart',
+        templateUrl: 'js/cart/cart.html',
+        controller: 'CartCtrl',
+    });
+});
+
+app.controller('CartCtrl', function($scope){
+	
+    $scope.items = JSON.parse(localStorage.getItem('cart')) || [];
+
+    $scope.removeItem = function(id, items) {
+        items = items.filter(function(elem) {
```

Comment:
I passed the items from the from the html page in as an argument, but I can probably refactor this to skip this step.